docs_src/callbacks.csv_logger.ipynb

```diff
@@ -32,7 +32,7 @@
       "text/markdown": [
```

Comment:
<p>Can you use the path variable here, with the function `os.remove` maybe? Notebooks are automatically executed as parts of our tests, so all the cells need to be able to execute anywhere.</p>

 _Reply via <a href='https://app.reviewnb.com/fastai/fastai/pull/1833/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>Sure will do that!</p>